test/extensions/wasm/test_data/stats_cpp.cc

```diff
@@ -72,4 +72,8 @@ extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onLog(uint32_t /* context_zero */) {
   auto complete_h = base_h->resolveAndExtend<std::string, bool>(7, "string_tag", "bool_tag");
   auto simple_h = complete_h->resolve("test_tag", true);
   logError(std::string("h_id = ") + complete_h->nameFromIdSlow(simple_h.metric_id));
+
+  Counter<std::string, int, bool> stack_c("test_counter", "string_tag", "int_tag", "bool_tag");
+  stack_c.increment(1, "test_tag_stack", 7, true);
+  logError(std::string("stack_c = ") + std::to_string(stack_c.get("test_tag_stack", 7, true)));
```

Comment:
Add `Gauge` and `Histogram` for coverage?

Reply:
Done.